src/Resources/TransactionTypes/Authorization.php

```diff
@@ -280,7 +280,7 @@ public function cancel($amount = null): Cancellation
     /**
      * Charge authorization.
      *
-     * @param null $amount
+     * @param float $amount
```

Comment:
```suggestion
     * @param float|null $amount
```

Reply:
I adjusted this, and for the rest of that class to make it consistent for that class.